files/httpRequest.tsf

```diff
@@ -1,14 +1,8 @@
 // AUTO_GENERATED Do not change this file directly change config.ts file instead
 import { AxiosRequestConfig, AxiosResponse } from "axios";
 //@ts-ignore
-import qs from "qs";
-import { getBaseConfig, getAxiosInstance } from "./config";
 
-function paramsSerializer(param: any) {
```

Comment:
Do not remove paramsSerializer. We added that because Axios has a problem with array prams. Maybe we can add an option to swagger.config for that in the future.

Reply:
I added paramSerializer in the baseConfig instead, now that there is a common instance for all axios requests